### PR TITLE
lab2 done

### PR DESCRIPTION
**Pull request by Malionkin**

### CODE REVIEW

File: lab2/Laba2.cpp [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        sum = sum + arr[i];                                  // нахожу сумму всех элементов между крайними нулями
+    }
+
+    if (sum > 0 )
+    {
+        cout << sum << endl;
+    }                                                           // если есть два несоседних нулевых элемента, между которыми есть что считать (элементы мужду нулями не являются нулями)
+    else                                                         // то вывожу значение суммы
+    {
+        cout << "Net nulevih elementov ili iskomaia summa ravna 0!!"<< endl;    //если условие не выполняется, вывожу буковки
```
Comment (Malionkin) [inline lab2/Laba2.cpp:111]: Write in english PLEASE!

File: lab2/Laba2.cpp [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+                cout << "Vvedite element " << i+1 << endl;
+                cin >> element;
+                arr[i] = element;
+            }
+            break;
+        }
+        case 2:
+        {
+            int span;
+            cout << "Zadayte granitsu intervala randomnyh chisel!!" << endl;
```
Comment (Merilian) [inline lab2/Laba2.cpp:46]: I want to choose interval boundaries from both sides. Like [9, 56]

File: lab2/Laba2.cpp [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+            {
+
+                arr[i] = rand()%span;
+            }
+            break;
+
+        }
+    }
+
+    int number = 1;
```
Comment (Merilian) [inline lab2/Laba2.cpp:58]: You do not need this variable

File: lab2/Laba2.cpp [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+    }
+
+    cout << "Teper' vypolniaem taski!" <<  endl;
+
+    int multi;
+    for (int i = 0; i < size; i++)
+    {
+        if (i%2 == 1)
+        {
+            multi = arr[i]*arr[i-2];                         // произведение четных элементов
```
Comment (Merilian) [inline lab2/Laba2.cpp:72]: This is not the task solution at all

File: lab2/Laba2.cpp [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+
+    for (int i = 0; i < size; i++)
+    {
+        if (arr[i] == 0)
+        {
+            min = i;                                           // нахожу первый нулевой элемент
+            break;
+        }
+    }
+    for (int i = size; i > 0; i--)
```
Comment (Merilian) [inline lab2/Laba2.cpp:89]: You not checking first element, but somehow it is OK (:

File: lab2/Laba2.cpp [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+    if (sum > 0 )
+    {
+        cout << sum << endl;
+    }                                                           // если есть два несоседних нулевых элемента, между которыми есть что считать (элементы мужду нулями не являются нулями)
+    else                                                         // то вывожу значение суммы
+    {
+        cout << "Net nulevih elementov ili iskomaia summa ravna 0!!"<< endl;    //если условие не выполняется, вывожу буковки
+    }
+
+for ( int o = 0; o < maxSize; o++)             // почему использую константу, чтобы обеспечить сколь нужно большое количество итераций цикла P.S. лень расписывать подробно, зачем я это сделал, если возникнут вопросы, напишу/отвечу лично
```
Comment (Merilian) [inline lab2/Laba2.cpp:114]: I don't need any comments at all :)
But in this case you can determine exactly how much iterations you need

**Pull request merged**